<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSub">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>sub</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMul">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>mul</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDiv">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>div</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPow">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>pow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSqrt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="edtZahl1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="edtZahl2">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtLog">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="edtZahl3">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>60</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="edtZahl4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSumme">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Summe</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
